pybaseball/datahelpers/statcast_utils.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import pandas as pd
+from typing import Optional
+
+def add_spray_angle(df: pd.DataFrame, adjusted: Optional[bool] = False):
```

Comment:
Please specify return type:

```suggestion
def add_spray_angle(df: pd.DataFrame, adjusted: Optional[bool] = False) -> pd.DataFrame:
```

As well, `Optional` is only needed of `None` is a valid value to pass to `adjusted`. Otherwise you can just leave it as `bool`

Reply:
Thanks - I've not used type hinting a ton, and actually just learned I was misusing Optional a couple of hours ago :) resolved in the latest commit. Thanks for the check.